internal/storage/bucket_handle.go

```diff
@@ -96,3 +98,38 @@ func (b *bucketHandle) StatObject(ctx context.Context, req *gcs.StatObjectReques
 
 	return
 }
+
+func (b *bucketHandle) CopyObject(ctx context.Context, req *gcs.CopyObjectRequest) (o *gcs.Object, err error) {
+	srcObj := b.bucket.Object(req.SrcName)
+	dstObj := b.bucket.Object(req.DstName)
+
+	// Switching to the requested Generation of Source Object.
```

Comment:
why are some of the words in capital letter here - Generation, Source, Object. please fix.

Reply:
fixed